Crash when tapping on push notification about starter pack arrival

# Bug Report


## Problem 
When user `Accept` the starter pack, PN "Your starterpack is on your way" comes.
At attempt to tap on it got an error: 

![IMAGE 2021-10-19 16:50:04](https://user-images.githubusercontent.com/4557972/137935121-9812b6ba-9da0-4b95-ad81-6002055bfcab.jpg)
Tested on `Rinkeby` network

#### Expected behavior


#### Actual behavior


### Reproduction

1) Install build, create user with enabled PNs, and switch to Rinkeby
2) Create referral link using these [steps](https://ethstatus.testrail.net/index.php?/tests/view/2331312&group_by=cases:section_id&group_order=asc&group_id=894)  or use `status-im://referral/3b16b2659a5c876985ba`)
3) Make QR from link, scan it and open in Status
4) After login will be redirected to 1-1 chat, tap on `Accept`
5) Pay attention to PN center 
6) Tap on PN `Starterpack is on your way` (approximate wording) 

### Additional Information

- Status version: RC 2 1.17 
- Operating System: Android, iOS

Logs: [Status-debug-logs (8).zip](https://github.com/status-im/status-react/files/7374387/Status-debug-logs.8.zip)

```
10-20 19:43:04.121 13068 13068 E AndroidRuntime: java.lang.RuntimeException: Error receiving broadcast Intent { act=im.status.ethereum.module.TAP_NOTIFICATION flg=0x10 (has extras) } in im.status.ethereum.pushnotifications.PushNotificationHelper$1@84ffe9
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:1407)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.app.-$$Lambda$LoadedApk$ReceiverDispatcher$Args$_BumDX2UKsnxLVrE6UJsJZkotuA.run(Unknown Source:2)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:873)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:201)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6831)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:927)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: Caused by: java.lang.NullPointerException: uriString
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.net.Uri$StringUri.<init>(Uri.java:490)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.net.Uri$StringUri.<init>(Uri.java:480)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.net.Uri.parse(Uri.java:452)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at im.status.ethereum.pushnotifications.PushNotificationHelper.getOpenAppIntent(PushNotificationHelper.java:109)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at im.status.ethereum.pushnotifications.PushNotificationHelper$1.onReceive(PushNotificationHelper.java:125)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0(LoadedApk.java:1397)
10-20 19:43:04.121 13068 13068 E AndroidRuntime: 	... 8 more
```

--- a/modules/react-native-status/android/src/main/java/im/status/ethereum/pushnotifications/PushNotificationHelper.java
+++ b/modules/react-native-status/android/src/main/java/im/status/ethereum/pushnotifications/PushNotificationHelper.java
@@ -106,7 +106,9 @@ public class PushNotificationHelper {
         //flags in this link:
         //https://stackoverflow.com/questions/52390129/android-intent-setflags-issue
         intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TOP);
-        intent.setData(Uri.parse(deepLink));
+        if (deepLink != null) {
+            intent.setData(Uri.parse(deepLink));
+        }
         return intent;
     }
 

